### ISSUE TITLE
Feature request: add option to disable wrap-around

### ISSUE THREAD

**Issue by aspiers** [raxod502/ctrlf#68]
Given that `M-<` and `M->` allow easy jumping to the first/last matches, I'd personally prefer if `ctrlf-{forward,backward}-*` didn't wrap around from last to first match and vice-versa.  Currently I find it too easy to press repeatedly (say) `C-s` and wrap around when I didn't want to.  Any chance of having a `defcustom` for disabling this behaviour?

**Comment by raxod502** [comment]
Yep, that sounds totally reasonable. And I think the implementation will not be too hard as some of the relevant primitives already exist; for example, the `:bound 'wraparound` argument passed to `ctrlf--search` currently could be changed to something conditional.

**Comment by jdtsmith** [comment]
I used to wait for the `(ding)` when reaching the end of the buffer; might be nice to optionally re-enable that.

**Comment by raxod502** [comment]
I think that's a fine idea as well, thanks for the suggestion!

**Comment by kings2u** [comment]
I'll cast my vote in favor of this feature. Personally I wouldn’t mind still being able to wrap around but at least getting a `(ding 1)` sent to visually signal when the wrapping occurs. Thank you for this tremendous package!

**Comment by kimim** [comment]
I vote for this. Make a warning before next wraparound is better.
Or even better, when I am at the end of the buffer, C-s will locate to the nearest match backward.
